lms/djangoapps/edxnotes/helpers.py

```diff
@@ -218,7 +218,7 @@ def preprocess_collection(user, course, collection):
                 if not chapter:
                     log.debug(u"Chapter not found: %s", usage_key)
                     continue
-                if chapter in cache:
+                if chapter in list(cache):
```

Comment:
@feanil  this fixes the test not hashable error here but on line 239 we are assinging values in dict. where a non hashable object is set as a key. thats where code breaks. Any ideas how to fix that.